src/Tests/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
SDK things?

Reply:
Yep -- this is set to false in all test projects.

One thing I did differently here is remove the config to generate a NuGet package from the tests since it was causing some long path weirdness. The tests are still running on Helix so I don't think that was totally needed.